docs/gitbook/tutorials/skipper-progressive-delivery.md

```diff
@@ -0,0 +1,382 @@
+# Skipper Canary Deployments
+
+This guide shows you how to use the [Skipper ingress controller](https://opensource.zalando.com/skipper/kubernetes/ingress-controller/) and Flagger to automate canary deployments.
+
+![Flagger Skipper Ingress Controller](https://raw.githubusercontent.com/weaveworks/flagger/master/docs/diagrams/flagger-skipper-overview.png)
+
+## Prerequisites
+
+Flagger requires a Kubernetes cluster **v1.14** or newer and Skipper ingress **0.11.40** or newer.
+
+Install Skipper ingress-controller using [upstream definition](https://opensource.zalando.com/skipper/kubernetes/ingress-controller/#install-skipper-as-ingress-controller).
```

Comment:
What about Flagger? I think we should tell people to install Flagger with:
```
kustomize build https://github.com/weaveworks/flagger/kustomize/kubernetes | kubectl apply -f -
```

Reply:
thanks - added